js/admin/src/components/BazaarPageHeader.js

```diff
@@ -65,7 +70,9 @@ export default class BazaarPageHeader extends Component {
                 Button.component({
                     className: 'Button Button--icon Connected',
                     icon: 'dashboard',
-                    onclick: () => window.open(flagrowHost + '/home')
```

Comment:
This variable is not defined when I click the button

`ReferenceError: flagrowHost is not defined`